database.json

```diff
@@ -992,12 +986,6 @@
     "repo": "deno-nessie",
     "desc": "A database migration tool for deno."
   },
-  "nessie": {
-    "type": "github",
-    "owner": "halvardssm",
-    "repo": "deno-nessie",
-    "desc": "A database migration tool for deno."
-  },
```

Comment:
don't remove these

Reply:
They are duplicate keys though; is this intended?

![image](https://user-images.githubusercontent.com/59528510/82076020-d9902080-96e5-11ea-83d5-75ae2fbab84f.png)
![image](https://user-images.githubusercontent.com/59528510/82076000-d09f4f00-96e5-11ea-908a-fffae5648026.png)

Reply:
not related to this change

Reply:
Duplicates removed in #497